.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,9 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
```

Comment:
So had another thought here :smile:

I wonder if we should have a little section here about `Dockerfile`-specific issues, where we link to the buildkit new issues page :thinking:

Reply:
Hum not sure how to phrase this, let's see in a follow-up but that sounds like a good idea.